tools/check_output_test.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# Usage: check_output_test.py [path to deno executable]
```

Comment:
* Add a coarse explanation (comment) about what the purpose of check_output_test.py is. It's not super obvious.
* Also, do you expect to do this "check_output" thing for every (future) test? That's not super common - I think many tests won't output anything, and if they exit with code 0 it means they succeeded.